parcelgen.py

```diff
@@ -199,7 +208,7 @@ def needs_jsonutil(self):
         if "Date" in self.props:
             return True
         for key in self.props.keys():
-            if "List" in key:
+            if "List" or "Map" in key:
```

Comment:
I don't think this syntax works. The following prints "Hello"

```python
if 5 or 6 in [1,2,3]:
     print "Hello"
```

I think the simplest thing you want is:
```python
if "List" in key or "Map" in key:
   return True
```